latex2sympy.py

```diff
@@ -768,6 +782,38 @@ def handle_exp(func):
     return sympy.exp(exp_arg)
 
 
+def handle_gcd_lcm(f, args):
+    """
+    Return the result of gcd() or lcm(), as UnevaluatedExpr
+
+    f: str - name of function ("gcd" or "lcm")
+    args: List[Expr] - list of function arguments
+    """
+
+    def apply_nested(f, lst):
+        """
+        sympy's gcd() and lcm() only support 2 parameters
+        this function calculates gcd() and lcm() for as many parameters as passed
+        """
+        if not lst:
+            # or modify this function and return the line below
```

Comment:
@tirkarp I'm not sure what these comments mean

Reply:
to be honest I'm not sure either 😂